packages/rancher-istio/charts/values.yaml

```diff
@@ -51,3 +51,27 @@ global:
     image: proxyv2
   proxy_init:
     image: proxyv2
+
+# this can be removed in 1.7 as it is default
+meshConfig:
+  enablePrometheusMerge: true
+
+# Kiali subchart from rancher-kiali-server
+kiali:
+  enabled: true
+  auth:
+    # todo: what auth strategy ?
+    # strategy: token # this is default, you can use a servic-account or kubeconfig token
+    strategy: anonymous
+  deployment:
+    ingress_enabled: false
+  external_services:
+    prometheus:
+      custom_metrics_url: "http://monitoring-rancher-monitor-prometheus.monitoring-system.svc:9090"
+      url: "http://monitoring-rancher-monitor-prometheus.monitoring-system.svc:9090"
```

Comment:
We should add a comment here (or to the `README.md` for this field) that this default depends on `rancher-monitoring` also being installed with the default values for `nameOverride`, `namespaceOverride`, and the `prometheus.service.port`.

If modified on the `rancher-monitoring` deployment, the appropriate value here should be:
`http://{{ .Values.nameOverride }}-prometheus.{{ .Values.namespaceOverride }}.svc:{{ prometheus.service.port }}` based on the values set on the `rancher-monitoring` chart.

Reply:
added